modules/distribution/src/repository/resources/conf/infer.json

```diff
@@ -115,5 +115,37 @@
     "symmetric": {
       "encryption.internal_crypto_provider": "org.wso2.carbon.crypto.provider.SymmetricKeyInternalCryptoProvider"
     }
+  },
+
+  "identity.auth_framework.endpoint.mutual_ssl_manager_enabled": {
+    "true": {
+      "identity.auth_framework.endpoint.hostname_verification_enabled" : true,
+      "identity.auth_framework.endpoint.mutual_ssl_username" : "admin",
+      "identity.auth_framework.endpoint.client_keystore" : "$conf{keystore.tls.file_name}",
+      "identity.auth_framework.endpoint.carbon_security_keystore_password" : "$conf{keystore.tls.password}",
+      "identity.auth_framework.endpoint.client_truststore" : "$conf{truststore.file_name}",
+      "identity.auth_framework.endpoint.carbon_security_truststore_password" : "$conf{truststore.password}",
+      "identity.auth_framework.endpoint.identity_server_service_url" : "https://localhost:9443/services/",
```

Comment:
Host and port should use the placeholders

Reply:
Fixed.